fluid/DeepASR/data_utils/data_reader.py

```diff
@@ -133,6 +134,9 @@ class DataReader(object):
         shuffle_block_num (int): Block number indicating the minimum unit to do 
                                  shuffle.
         random_seed (int): Random seed.
+        verbose (int): Whether to suppress the complaints from sub-process. If
```

Comment:
Why don't use ```bool```?

Reply:
We may add more verbose level for debugging.

Reply:
Then please modify the comments, don't use ```Whether to ...```